k8s/namespaces/private-law/prl-cos/ithc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: prl-cos
+  namespace: private-law
+spec:
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/prl/cos:prod-4bccd66-20220516112343 #{"$imagepolicy": "flux-system:prl-cos"}
```

Comment:
`image:` not needed in any of the {env}.yaml files if it's just using the base image from the prl-cos.yaml file

Reply:
Addressed and removed as it is there with the prl-cos.yaml file